Add more Decentlab devices

decentlab opened the pull request:
<!--
Thanks for submitting a pull request. Please fill the template below,
otherwise we will not be able to process this pull request.
-->

#### Summary
Adds all remaining 37 devices

#### Changes
<!-- What are the changes made in this pull request? -->

- Adds:
  + dl-5tm
  + dl-atm22
  + dl-atm41
  + dl-ctd10
  + dl-dlr2-002
  + dl-dlr2-003
  + dl-dlr2-004
  + dl-dlr2-005
  + dl-dlr2-006
  + dl-dlr2-008
  + dl-dlr2-010
  + dl-dlr2-012
  + dl-ds18
  + dl-dws
  + dl-itst
  + dl-kl66
  + dl-lid
  + dl-lp8p
  + dl-optod
  + dl-par
  + dl-pm
  + dl-pr21
  + dl-pr36
  + dl-pr36ctd
  + dl-pyr
  + dl-rhc
  + dl-sht35-001
  + dl-sht35-002
  + dl-smtp
  + dl-tbrg
  + dl-trs11
  + dl-trs12
  + dl-trs21
  + dl-wrm
  + dl-zn1
  + dl-zn2

- Fixes dl-mbx

johanstokking reviewed (approved):
Thanks! I added the sensors to the right place, otherwise no changes were necessary

decentlab commented:
> Thanks! I added the sensors to the right place. Also I removed `4-20 ma`; let's use `current` and add scale later.

Thanks @johanstokking for reviewing all of it. Can we in some way include `4-20 ma`? Because this is one of our more important sensor products. Many people look for this specific interface. This is like UART or SDI-12. Thanks very much.

johanstokking commented:
@decentlab OK I understand. I added `4-20 ma` now. Please rebase any ongoing work on the updated `master` branch.
